tutorials/preprocessing/plot_20_rejecting_bad_data.py

```diff
@@ -69,7 +69,6 @@
 # functions or class constructors, including:
 #
 # - creation of epoched data from continuous data (:class:`mne.Epochs`)
-# - independent components analysis (:class:`mne.preprocessing.ICA`)
```

Comment:
Instead of deleting it should probably be "many methods of ..." since it is in many places here, including probably the most import (`fit`):

https://mne.tools/dev/generated/mne.preprocessing.ICA.html